src/recon_buildblock/distributable.cxx

```diff
@@ -570,4 +576,140 @@ void distributable_computation(
        % CPU_timer.value() % wall_clock_timer.value());
 }
 
+void LM_distributable_computation(
+        const shared_ptr<ProjMatrixByBin> PM_sptr,
+        const shared_ptr<ProjDataInfo>& proj_data_info_sptr,
+        DiscretisedDensity<3,float>* output_image_ptr,
+        const DiscretisedDensity<3,float>* input_image_ptr,
+        const std::vector<Bin>& record_ptr,
+        int subset_num, int num_subsets,
+        const std::vector<float>* additive_binwise_correction)
+{
+
+    CPUTimer CPU_timer;
+    CPU_timer.start();
+    HighResWallClockTimer wall_clock_timer;
+    wall_clock_timer.start();
+
+    assert(!record_ptr.empty());
+
+    const float max_quotient = 10000.F;
+
+    if (output_image_ptr != NULL)
+      output_image_ptr->fill(0.F);
+
+std::vector< shared_ptr<DiscretisedDensity<3,float> > > local_output_image_sptrs;
+std::vector<double> local_log_likelihoods;
+std::vector<int> local_counts, local_count2s;
+std::vector<Bin> local_measured_bin, local_basic_bin, local_fwd_bin;
+std::vector<ProjMatrixElemsForOneBin> local_row;
+std::vector<float>measured_div_fwd;
+#ifdef STIR_OPENMP
+#pragma omp parallel shared(local_output_image_sptrs,local_row, local_log_likelihoods, local_counts, local_count2s, local_measured_bin, local_fwd_bin)
+#endif
+    // start of threaded section if openmp
+    {
+#ifdef STIR_OPENMP
+#pragma omp single
+        {
+            std::cerr << "Starting loop with " << omp_get_num_threads() << " threads\n";
+            local_output_image_sptrs.resize(omp_get_max_threads(), shared_ptr<DiscretisedDensity<3,float> >());
+            //            local_log_likelihoods.resize(omp_get_max_threads(), 0.);
+            local_counts.resize(omp_get_max_threads(), 0);
+            local_count2s.resize(omp_get_max_threads(), 0);
+            measured_div_fwd.resize(omp_get_max_threads(), 0.f);
+            local_measured_bin.resize(omp_get_max_threads(), Bin());
+            local_basic_bin.resize(omp_get_max_threads(), Bin());
+            local_fwd_bin.resize(omp_get_max_threads(), Bin());
+            local_row.resize(omp_get_max_threads(), ProjMatrixElemsForOneBin());
+        }
+
+#pragma omp for schedule(runtime)
```

Comment:
```suggestion
#pragma omp for schedule(dynamic)
```
`runtime` means depend on an env variable, see https://610yilingliu.github.io/2020/07/15/ScheduleinOpenMP/#runtime, and this defaults to slow code on MacOS.